### ISSUE TITLE
Migrate kpt to standalone github org

### ISSUE THREAD

**Issue by mortent** [GoogleContainerTools/kpt#4034]
As part of the process to donate kpt to CNCF, we need to move the kpt repos from GoogleContainerTools to a standalone github org.

**Comment by kispaljr** [comment]
Maybe this is not the best place to ask this, but:

Are you planning to also donate the  https://github.com/GoogleContainerTools/kpt-functions-sdk repository to CNCF together with the rest of kpt? 
(And move the repo to the new org, and change the CLA?)

**Comment by mortent** [comment]
@kispaljr Yes, we plan on migrating all of the kpt repos into the new org. The list is provided in the CNCF application: https://github.com/cncf/sandbox/issues/34

**Comment by kispaljr** [comment]
Do you have an expected date of finishing the migration?
